### ISSUE TITLE
Change all references from sway, to swayfx

### ISSUE THREAD

**Issue by sonjiku** [wlrfx/swayfx#191]
Hey,

I wanna try this fork on fedora 38. Installing this though, means I have to uninstall sway, sway-config-upstream, and a lot of other dependencies and unused packages that the official sway pkg relies on, and I need.

The official sway maintainers have stated multiple times that they don't plan on implementing the features you have added, and don't plan to merge a fork like this. 

I suggest just renaming all references in code from "sway" to "swayfx", or any name of your choosing. This will mean that people using sway as their daily driver, don't have to uninstall it *just to **try*** this project out. and both compositors can live happily in a system.

**Comment by ErikReider** [comment]
What you could do in the meantime is to build it manually and run that executable in a tty while still having sway installed.

```
git clone https://github.com/WillPower3309/swayfx.git swayfx && cd swayfx
meson build -Dwerror=false
meson compile -C build
```

And run `./build/sway/sway` in a tty.

**Comment by werdahias** [comment]
Probably related: I looked into building a debian package for swayfx,  but the binaries and manpages conflict with the regular sway package. I could hack around this by renaming` /usr/bin/sway` to` /usr/bin/swayfx` and so on but it would really simplify adaption if all refererences to sway were renamed,

**Comment by MrPenguin07** [comment]
I concur with renaming all references to sway, allowing both to be installed at the same time - with separate configs for both.

Sway have made their bed re. hard refusal to modernize and accomodate a growing number of users wanting simple effects, this fork would benefit from being more independent.

**Comment by bhepple** [comment]
I came here to say exactly this - thanks to OP for the fluent report.

**Comment by markstos** [comment]
I disagree here. I've been been switching back and forth between Sway and SwayFX when debugging either way.

I find it to be a *feature* that they have same binary and conflict. That's because there are several tools to that use `sway` or `swaymsg` and all these would break if the name changed. At least on Arch, attempting to install the other will prompt to install the conflicting one-- easy.

If it's going to be renamed, then there should be another package with a name like `swayfx-sway-dropin` which simply symlinks to swayfx to `sway` so that it's easy to have `swayfx` installed, but have all the things that expect `sway/swaymsg` to be installed to still work.
